tests/examples/pacer/case_queries/insb_1.json

```diff
@@ -12,5 +12,10 @@
   "date_last_filing": "2016-06-28",
   "date_terminated": "2016-06-28",
   "docket_number": "13-07422",
+  "federal_defendant_number": "7",
```

Comment:
This looks wrong. `13-07422-RLM-7A` is the docket number string that is parsed - but here I think it's 7 for Chapter 7 bankruptcy. so there should not be a defendant number.  

I think we should update the regex to perhaps include \b a boundary or whitespace to parse out the end?

Reply:
Also we should maybe separate out the bankruptcy parser completely from the district parser?  that might make this less likely to occur.

Reply:
I split the regex for district and bankruptcy so wrong parsing, as in the example can be avoided. Considering because bankruptcy numbers won't have `federal_defendant_number` and `federal_dn_judge_initials_referred`.